GamingTools/PlaytimeEstimator/PlaytimeEstimator1.1.bas

```diff
@@ -0,0 +1,337 @@
+' Playtime Estimator based on Capture Images / Version 1.1 (Refactoring)
+'
+' Date   : 2025.7.19.
+' Author : kimpro82
+'
+' This VBA module scans directories, collects file information, prints the data to the worksheet, and calculates playtime statistics.
+
+
+Option Explicit
+
+
+' Structure to hold file information
+Private Type FileInfo
+
+    fileName                As String   ' Name of the file
+    fileType                As String   ' File type description
+    fileSize                As Integer  ' File size in bytes
+    fileDateLastModified    As Date     ' Last modified date
+
+End Type
+
+
+' Constants
+Const CELL_PATH_INPUT   As String = "B2"    ' Path list input starts here
+Const CELL_OUTPUT_START As String = "A11"   ' Output data starts here
+Const CELL_CALC_START   As String = "F3"    ' Calculation area starts here
+
+Const MAX_PATH          As Integer = 5      ' Maximum number of paths to scan
+Const MAX_ROW           As Long = 10000     ' Maximum number of rows and columns for output area
+Const MAX_COL           As Long = 12
+
+Private Const PLAYTIME_TERM_COUNT As Integer = 4
+Private PLAYTIME_TERMS(1 To PLAYTIME_TERM_COUNT) As Double
```

Comment:
There are still many occurrences of `4` as a magic number, not replaced with `PLAYTIME_TERM_COUNT`.

Reply:
But I'm tired ……